helm/secrets-provider/templates/secrets-provider.yaml

```diff
@@ -29,12 +29,6 @@ spec:
         imagePullPolicy: {{ .Values.secretsProvider.imagePullPolicy }}
         name: {{ .Values.secretsProvider.name }}
         env:
-        - name: SECRETS_DESTINATION
-          value: k8s_secrets
-
-        - name: CONTAINER_MODE
-          value: application
```

Comment:
why was this removed?

Reply:
wasn't removed, it was pushed below with all the other configs